src/main/scala/firrtl/ir/IR.scala

```diff
@@ -15,9 +15,12 @@ abstract class FirrtlNode {
   def serialize: String
 }
 
-abstract class Info extends FirrtlNode {
-  // default implementation
+/** Use the [[firrtl.ir.Serializer]] to serialize this node. */
+trait UseSerializer extends FirrtlNode {
```

Comment:
```suggestion
private[firrtl] trait UseSerializer extends FirrtlNode {
```
There's no value in external APIs extending this since it will crash anyway

Reply:
good catch!

Reply:
fixed